src/styles.css

```diff
@@ -55,12 +56,100 @@ select {
   padding-right: 5%;
 }
 
+@media screen and (max-width: 600px) {
```

Comment:
I believe we can avoid the !important flag on every attribute by putting this media query at the bottom of our CSS file instead of the top, in which case it would be the last set styling for every attribute and therefore the prioritized option. 

I tested this out on a pulled down version of your branch and it seemed to keep all the same styling without any !important flags for me. Can you please verify that as well on your side?

Reply:
Thanks Jake for catching that! I did not know you could do that. Also there was just one important flag I needed to keep on there.